src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/GraphNodes/Actions/CheckChildrenGraphActionHandler.cs

```diff
@@ -56,14 +56,10 @@ public override bool HandleRequest(IGraphContext graphContext)
                     continue;
                 }
 
-                if (dependency.Flags.Contains(DependencyTreeFlags.SupportsHierarchy))
-                {
-                    trackChanges = true;
```

Comment:
Yes, will remove it.

---
In reply to: [111636175](https://github.com/dotnet/project-system/pull/1991#discussion_r111636175) [](ancestors = 111636175)